### PR TITLE
Create an example notebook to show users how to use the pipeline

### PR DESCRIPTION
**Pull request by paddyroddy**
PR authored by @nickbearman, but I'm making a draft PR because I have merged #16 and want to make sure I know what is being changed here.

### CODE REVIEW

**Comment by paddyroddy** [comment]
Two main thoughts @nickbearman.

1. Should we be committing the `.mps` file? I don't know what it does.

2. Can you re-create the journeys/failures files now that I've updated the `main`. Probably including car etc in too.

**Comment by paddyroddy** [comment]
Also the map now renders in the notebook

File: README.md [hunk 1]
```diff
@@ -1,17 +1,50 @@
...
+- [Short Paper](agile/short-paper.pdf)
+
+Currently (as at 06/06/2023) the new code developed for `spopt` is not integrated in to the main `spopt` version. There is currently a PR in progress for this at https://github.com/pysal/spopt/pull/374.
+
+## Installation of pre-processing code
+
+Install using `pip`:
+
+```python
+python -m pip install --upgrade pip
```
Comment (paddyroddy) [inline README.md]: You don't _really_ need to upgrade `pip`. That was a specific issue with your system.

File: README.md [hunk 2]
```diff
@@ -1,17 +1,50 @@
...
+```
+
+Then re-run. You can check if it’s worked by running
+`echo $TFL_APP_KEY`, and `export N_CORES=1`.
+
+Run using
 
 ```sh
-tfl example_subject
+tfl data example_subject
```
Comment (paddyroddy) [inline README.md]: You need to change this
```suggestion
tfl example_subject
```

**Comment by nickbearman** [comment]
> Two main thoughts @nickbearman.
> 
> 1. Should we be committing the `.mps` file? I don't know what it does.

I think this is created when the model is run. Given we also now have a CSV output, I think we can treat this as a temporary file and ignore it. 

> 2. Can you re-create the journeys/failures files now that I've updated the `main`. Probably including car etc in too.

Yes I think so

File: README.md (hunk 1 above)
Comment (nickbearman) [reply README.md]: Happy for you to keep or remove as you see fit

File: README.md (hunk 2 above)
Comment (nickbearman) [reply README.md]: Fine with me

**Comment by paddyroddy** [comment]
I disagree about the `.mps` file unless it is much slower without it. I will investigate. In general, we should only have stuff in a repo that is essential.

**Pull request merged**